pylint/pyreverse/main.py

```diff
@@ -146,6 +155,7 @@
             dest="colorized",
             action="store_true",
             default=False,
+            metavar="<y or n>",
```

Comment:
This is also a flag and just used as ``--colorized``, so the metavar does not really fit here.